clang-tools-extra/test/clang-tidy/bsl-temp.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %check_clang_tidy %s bsl-for-loop %t
+
+// A6-5-2
+void foo()
+{
+	int y = 0;
```

Comment:
Make sure you run clang-format so that all of the PRs use LLVM's formatting rules.